packages/relay-kit/src/packs/safe-4337/types.ts

```diff
@@ -184,25 +177,35 @@ type UserOperationStringValues = Omit<
   maxPriorityFeePerGas: string
 }
 
-export type PimlicoCustomRpcSchema = [
+export type Safe4337RpcSchema = [
   {
-    Method: 'pimlico_getUserOperationGasPrice'
-    Parameters: never
-    ReturnType: {
-      slow: { maxFeePerGas: string; maxPriorityFeePerGas: string }
-      standard: { maxFeePerGas: string; maxPriorityFeePerGas: string }
-      fast: { maxFeePerGas: string; maxPriorityFeePerGas: string }
-    }
+    Method: RPC_4337_CALLS.GET_PAYMASTER_STUB_DATA
+    Parameters: [UserOperationStringValues, string, string, { token: string }?]
+    ReturnType:
+      | {
+          paymasterAndData: string
+        }
+      | {
+          paymaster: string
+          paymasterData: string
+          paymasterVerificationGasLimit?: string
+          paymasterPostOpGasLimit?: string
+        }
   },
   {
-    Method: 'pm_sponsorUserOperation'
-    Parameters: [UserOperationStringValues, string, { sponsorshipPolicyId?: string }?]
-    ReturnType: {
-      paymasterAndData: string
-      callGasLimit: string
-      verificationGasLimit: string
-      preVerificationGas: string
-    }
+    Method: RPC_4337_CALLS.GET_PAYMASTER_DATA
+    Parameters: [UserOperationStringValues, string, string, { token: string }?]
+    ReturnType:
```

Comment:
These are Pimlico specific methods. Why are they defined in the general types instead of Pimlico estimator specific types?